package/lbcamden/components/button/_index.scss

```diff
@@ -324,9 +307,9 @@ $button-shadow-size: $govuk-border-width-form-element;
     }
     vertical-align: middle;
     -ms-flex-negative: 0;
```

Comment:
Do we still need this prefixing?

Reply:
@alexwybraniec do you want to get rid of unnecessary prefixing throughout or just this instance?

Reply:
I'm curious if the prefixing is still needed if we are dropping IE8 support? Check caniuse for flexbox and see what it says

Reply:
Will do. 

Should we remove all instances of this kind of prefix in other places? They're used quite a lot.

Reply:
ie10 and 11 need the prefix, below that don't support flexbox.